Emu/.emu_setup/standard_launch.sh

```diff
@@ -62,6 +62,16 @@ if [ "$MODE" != "overclock" ] && [ "$MODE" != "performance" ]; then
 	/mnt/SDCARD/spruce/scripts/enforceSmartCPU.sh &
 fi
 
+##### SyncThing Sync Check, perform only once per session #####
```

Comment:
Let's standardize on one capitalization of syncthing. Pretty much everywhere else it's Syncthing.

Reply:
Lol fairrrrr, while building this feature, the first half of the time I thought this was SyncThing, whoopsies:

e6c3a1e00dcd746793baacc92ebecb7b207cd333